test/tests/functional/pbs_node_rampdown.py

```diff
@@ -6945,6 +6646,9 @@ def test_release_multi_jobs(self):
                             {'resources_available.ncpus': 3},
                             id=self.hostC)
 
+        # Wait for nodes to come up
+        time.sleep(10)
```

Comment:
instead of this, can you do an expect on node state = free to check that they are up?